src/app/find/errorReports/ErrorReportContainer.jsx

```diff
@@ -107,65 +110,67 @@ class ErrorReportContainer extends Component {
     }
 
     return (
-      <Router>
-        <div>
-          <Modal>
-            <div className="mx-3 mt-4 position-relative">
-              <Icon
-                name="times"
-                onClick={goToViewLocation}
-                style={{
-                  position: 'absolute',
-                  right: 0,
-                  top: '0.2em',
-                }}
-              />
-            </div>
-            <Switch>
-              <Redirect exact from={`${this.props.match.path}/`} to={`${this.props.match.path}/services`} />
-              <Route
-                exact
-                path={`${this.props.match.path}/text`}
-                render={() => (
-                  <ErrorReportText
-                    match={this.props.match}
-                    errorReportText={this.state.errorReportText}
-                    onChange={this.onErrorReportTextChanged}
-                    onSubmit={this.onErrorReportSubmitted}
-                    goToViewLocation={goToViewLocation}
-                  />
-                )}
-              />
-              <Route
-                exact
-                path={`${this.props.match.path}/thanks`}
-                render={() => (
-                  <Thanks
-                    goToViewLocation={goToViewLocation}
-                  />
-                )}
-              />
-              <Route
-                exact
-                name="servicesScreen"
-                path={`${this.props.match.path}/services`}
-                render={() => (
-                  <ErrorReportInformationSelect
-                    locationData={locationData}
-                    match={this.props.match}
-                    generalLocationError={errorReportGeneralLocationError}
-                    errorReportServices={errorReportServices}
-                    onServiceChange={this.onErrorReportServicesChanged}
-                    onGeneralLocationChange={this.onErrorReportGeneralLocationErrorChanged}
-                    goToViewLocation={goToViewLocation}
-                  />
-                )}
-              />
-              <Route path="*" component={NotFound} />
-            </Switch>
-          </Modal>
-        </div>
-      </Router>
+      <div>
+        <Modal>
+          <div className="mx-3 mt-4 position-relative">
+            <Icon
+              name="times"
+              onClick={goToViewLocation}
+              style={{
+                position: 'absolute',
+                right: 0,
+                top: '0.2em',
+              }}
+            />
+          </div>
+          <Switch>
+            <Redirect
+              exact
+              from={`${this.props.match.path}/`}
+              to={`${this.props.match.path}/services`}
+            />
+            <Route
+              exact
+              path={`${this.props.match.path}/text`}
+              render={() => (
+                <ErrorReportText
+                  match={this.props.match}
```

Comment:
No need to pass `match` anymore, right?
After all, the child components don't deal with navigation and URL concerns anymore.